public/cli/lib/rendering/recordFrames.ts

```diff
@@ -0,0 +1,49 @@
+const puppeteer = require('puppeteer-core');
+const chromium = require('chromium');
+const path = require('path');
```

Comment:
These should now use `import` instead of `require`.

Reply:
got 2 out of 3.  There is no @types/chromium so we'll leave it there for now and resolve later.